File: MY PROGRAMS/LOGIC BUILDING-13 ASSIGNMENTS/a8q5.c
/* Write a program which accept number from user and display its table in reverse order.

Input:5
Output:50 45 40 35 30 25 20 15 10 5

Input:-5
Output:50 45 40 35 30 25 20 15 10 5

Input:2
Output:20 18 16 14 12 10 8 6 4 2

*/




#include<stdio.h>

void Display(int ino)
{
	int i=0,imult=1;

	if(ino<0)
	{
		ino=-ino;
	}

	for(i=10;i>=1;i--)
	{
		imult=i*ino;

		printf("%d ",imult);	
	}
}

int main()
{
	int ivalue=0;

	printf("Enter the no.:");
	scanf("%d",&ivalue);

	Display(ivalue);

	return(0);
}

File: MY PROGRAMS/LB/JAVA/NUMBER/a8q5.java
/*	5. Write a program which accept number from user and display its table in reverse order.

	Input:	2
	Output:	20 18 16 14 12 10 8 6 4 2

	Input:	-5
	Output:	50 45 40 35 30 25 20 15 10 5

	Input:	5
	Output:	50 45 40 35 30 25 20 15 10 5	*/




import java.util.*;

class Assignment8
{
	public void Display(int ino)
	{
		int i=0,itable=1;

		if(ino<0)
		{
			ino=-ino;
		}

		System.out.println("Table is ");

		for(i=10;i>0;i--)
		{
			itable=ino*i;

			System.out.print(itable+"\t");
		}
	}
}

class a8q5
{
	public static void main(String arg[])
	{
		Scanner sobj=new Scanner(System.in);

		int ivalue=0;

		System.out.println("Enter the no.: ");
		ivalue=sobj.nextInt();

		Assignment8 aobj=new Assignment8();

		aobj.Display(ivalue);
	}
}
